src/components/adherence/summary/AdherenceAlerts.tsx

```diff
@@ -150,11 +154,9 @@ const AlertText: FunctionComponent<{alert: AdherenceAlert; studyId: string}> = (
 
 const AlertMenu: FunctionComponent<{
```

Comment:
make AlertMenu a 'dumb' component. it is clearer when the updates take place in the same place as data fetches so this component just tells it's parent what to do